### DISCUSSION TITLE
Game stuck on skip button as a result of broken mp3

### DISCUSSION THREAD

**Discussion by hrfarmer** [ppy/osu#17674] [Q&A] [answered]
So on this map ( https://osu.ppy.sh/beatmapsets/1624873#osu/3317510 ) the mp3 is broken for some reason, but the issue I have is that when I made a practice difficulty and tried to skip, the game ended up getting stuck, where the skip button did nothing and the pause menu did nothing after pausing and unpausing. I was still able to exit the map by opening the toolbar and clicking the home button
(also im not sure if this has been reported yet or not so just tell me if it has)

https://streamable.com/y1xo0b

[performance.log](https://github.com/ppy/osu/files/8422536/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/8422537/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/8422538/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/8422539/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/8422541/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/8422542/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8422543/updater.log)
[database.log](https://github.com/ppy/osu/files/8422544/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/8422545/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/8422546/network.log)
b

**Comment by peppy** [comment]
The logs you provided don't match the video.

I also have no idea what your "practice difficulty" involves. Please export and attach it.

**Reply by hrfarmer** [reply]
oh sorry about that

here is what should be the correct logs
[database.log](https://github.com/ppy/osu/files/8426500/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/8426502/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/8426504/network.log)
[performance.log](https://github.com/ppy/osu/files/8426505/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/8426506/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/8426507/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/8426508/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/8426509/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/8426510/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8426511/updater.log)


and here is the map with the practice difficulty i made
https://hrfarmer.s-ul.eu/uH5AORTT

**Reply by hrfarmer** [reply]
oops forgot to add recording 
https://streamable.com/gtjzqz

**Comment by bdach** [comment, marked as answer]
The "mp3" is broken because it is not in fact an mp3.

![2022-04-06-195648_595x550_scrot](https://user-images.githubusercontent.com/20418176/162038312-42519c89-8835-4d4e-afc1-ac1040e0a465.png)

```
$ file audio.mp3 
audio.mp3: ISO Media, MPEG v4 system, Dynamic Adaptive Streaming over HTTP
```

Please use an actual mp3 file and try again.
